packages/interactivity/src/utils/warn.ts

```diff
@@ -0,0 +1,21 @@
+const logged = new Set();
+
+export const warn = ( message ) => {
+	// @ts-expect-error
+	if ( typeof SCRIPT_DEBUG !== 'undefined' && SCRIPT_DEBUG === true ) {
```

Comment:
Flagging as something I'll need to update in #61486

Reply:
I'll move it to a `warn` function so it is not repeated everywhere.